lib/marqeta/api_object.rb

```diff
@@ -53,7 +53,7 @@ def created_time
     attr_accessor :attributes_hash
 
     def symbolized_attributes_hash
-      Hash[attributes_hash.map { |k, v| [k.to_sym, v] }]
+      Hash[attributes_hash.to_h.map { |k, v| [k.to_sym, v] }]
```

Comment:
What's the reason for this change?

Reply:
Oh duh. I see why.

Reply:
the deprecation is in the ticket description 😬